src/python/CRABClient/JobType/UserTarball.py

```diff
@@ -140,13 +142,13 @@ def upload(self, filecacheurl=None):
 
         self.close()
         archiveName = self.tarfile.name
-        archiveSizeKB = os.path.getsize(archiveName)/1024
+        archiveSizeKB = os.path.getsize(archiveName)//1024
         if archiveSizeKB <= 512 :
           archiveSize = "%d KB" % archiveSizeKB
         elif archiveSizeKB < 1024*10 :
           archiveSize = "%3f.1 MB" % (archiveSizeKB/1024.)
```

Comment:
I understand, but I fear more the danger of confusing me in the short term, if in the distant future someone who only knows python3 will wonder why someone felt the need to make it explicit that "/" indicates floating point division.. well they can clean up at that time !  I.e. I could remove the dot, but then put a comment, would that be better ?